google-datacatalog-rdbms-connector/README.md

```diff
@@ -189,11 +190,77 @@ and `_get_path_to_num_rows_query`.
 You can see working examples of implementing optional queries in the connectors code for PostgreSQL and MySQL. 
 Please also refer to these samples to see how user configuration file `ingest_cfg.yaml` should look like.
 
+### 4.2 Add support for SQL Objects
+You can use the SQL Objects mechanism from the SQL connector to ingest SQL objects such as Functions,
+Stored Procedures, Views, Materialized Views, and so on. This mechanism uses naming convention to locate
+a SQL query and a metadata definition file.
+
+You can see working examples of implementing SQL Objects in the test sources. 
+To enable it you need 3 configuration files:
+1. `ingest_cfg.yaml`
+    ```yaml
+    sql_objects:
+      - name: 'functions'
+        enabled: True
+      - name: 'stored_procedures'
+        enabled: True
+    ```
+
+    The `ingest_cfg.yaml` file must be located at the connection execution directory. You can find a sample in the [SAP HANA connector](../google-datacatalog-saphanaconnector/src/google/datacatalog_connectors/saphana/ingest_cfg.yaml).
+    
+    You can specify a list of sql objects with a flag to enable/disable it,
+    it will be only considered if the `enabled` flag is set to `True`.
+    
+    The `query` and `metadata_definition` files use the 
+    `sql_objects.name` value to be located, in the sample above,
+     for the `function` SQL Object two files are expected: 
+     `query_functions_sql_object.sql` and 
+    `metadata_definition_functions_sql_object.sql`.
+    
+    And the `stored_procedures` SQL Objects expects:
+    `query_stored_procedures_sql_object.sql` and 
+    `metadata_definition_stored_procedures_sql_object.sql`.
+    
+
+
+1.  [query_functions_sql_object.sql](tests/google/datacatalog_connectors/rdbms/test_data/query_functions_sql_object.sql)  
+    Implement a query file that matches the sql object name, with the following pattern:
```

Comment:
I believe it would be better in terms of readability if all **sql object** expressions were replaced with **SQL Object**. WDYT?